sdk/storage/CONTRIBUTING.md

```diff
@@ -1,187 +0,0 @@
-  | ACCOUNT_KEY               | Can be obtained from the Azure portal. Go to the `Access Keys` tab under the `Settings` section of your storage account in the portal and note down the key.                                                                                                                                                                                                                                                                                  | Required                                                                         |
-  | ACCOUNT_SAS               | Can be obtained from the Azure portal. Go to the `Shared access signature` tab under the `Settings` section of your storage account in the portal. Generate SAS by enabling permissions, setting exiry dates, etc. You can also generate SAS from tools like Azure Storage Explorer. <br>A SAS starts with "?". And if you are using Windows CMD, you may need quotes to escape special characters like this - `set "ACCOUNT_SAS=<YOUR_SAS>"` | Required to run the browser tests                                                |
-  | ACCOUNT_TOKEN             | Preferred way is to get a Token through Azure CLI.<br> - `az login`<br> - `az account get-access-token --resource "https://storage.azure.com"`<br>For more info - refer [Azure CLI - get-access-token](https://docs.microsoft.com/en-us/cli/azure/account?view=azure-cli-latest#az-account-get-access-token)                                                                                                                                  | Not required (Some of the tests will be skipped if this variable is not defined) |
-  | STORAGE_CONNECTION_STRING | Can be obtained from the Azure portal. Go to the `Access Keys` tab under the `Settings` section of your storage account in the portal and note down the connection string.                                                                                                                                                                                                                                                                    | Required                                                                         |
-  | ENCRYPTION_SCOPE_1        | An encryption scope created under your storage account. The two encryption scopes need to be different. Can be created using the SRP API.                                                                                                                                                                                                                                                                                                                                                     | Not required (Some of the tests will be skipped if this variable is not defined) |
```

Comment:
I'm ok to drop most of content in storage/contribution.md, but these environment variables are very helpful for storage devs. We need to document them somewhere.

Reply:
@XiaoningLiu you are right. I remember now this is the reason we still keep the `storage/CONTRIBUTING.md`.  I will revert this deletion.